tenant_schemas/postgresql_backend/introspection.py

```diff
@@ -5,6 +5,11 @@
 from django.db.backends.base.introspection import (
     BaseDatabaseIntrospection, FieldInfo, TableInfo,
 )
+try:
+    # Django >= 1.11
```

Comment:
also, according to https://docs.djangoproject.com/en/1.11/releases/1.11/#deprecating-warnings-are-no-longer-loud-by-default, we might skip the try...except and drop django < 1.11 for all future releases